internal/testing/alignment.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package testing
+package testing // import "go.opentelemetry.io/otel/internal/testing"
```

Comment:
While internal packages aren't exported, they are imported inside the code base, thus adding vanity import.